### DISCUSSION TITLE
Wifi not working

### DISCUSSION THREAD

**Discussion by tsarmark** [sebanc/brunch#995] [General]
Hello Sebanc, I have a query; I was trying to install ChromeOS on this device: hp stream 11 d001dx, in fact it installed almost fine as the only thing not working is the wifi; therefore, I cannot configure the laptop. Would there be some kind of solution? I am actually buying a wifi adapter, but I had that query, if there would be any solution. Thanks for the support Sebanc

**Comment by Buzzcut22** [comment]
Hi, 

I also encountered the  same problem on my hp laptop. it was a perfect install except for the wifi. I am  newbie so any step by step tutorial will be a great help. Thank you.
